packages/tasks/src/model-libraries.ts

```diff
@@ -700,6 +700,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.whisperkit,
 		countDownloads: `path_filename:"model" AND path_extension:"mil" AND _exists_:"path_prefix"`,
 	},
+	"3dtopia-xl": {
+		prettyLabel: "3DTopia-XL",
+		repoName: "3DTopia-XL",
+		repoUrl: "https://github.com/3DTopia/3DTopia-XL",
+		filter: false,
+		countDownloads: `path_extension:"pt"`,
```

Comment:
```suggestion
		countDownloads: `path:"model_vae_fp16.pt"`,
```

Better to count downloads  on a specific file. Since both `.pt` files are downloaded when a user uses the library, counting all `.pt` downloads will lead to duplicate counting.

Reply:
Thanks for the update!